bundle/manifests/kogito-serverless-operator.clusterserviceversion.yaml

```diff
@@ -28,6 +28,9 @@ metadata:
               "sw.kogito.kie.org/description": "Greeting example on k8s!",
               "sw.kogito.kie.org/version": "0.0.1"
             },
+            "labels": {
```

Comment:
@spolti @ricardozanini during the build the label added in the sample (sw.kogito_v1alpha08_kogitoserverlessworkflow_devmode.yaml) is applied here, I think is something to fix.